builder/dockerfile/internals_unix.go

```diff
@@ -1,3 +1,5 @@
+// +build darwin linux
+
```

Comment:
adding `interanals_darwin.go` instead of modifying this line would be nicer.

Reply:
Okay, atm internals_unix and internal_darwin has the same code. I believe this was done with an idea as this will be the code for the future work on `chown` functions.

Reply:
ah, then only rename (_linux.go=>_unix.go) should be fine (no need to add //+build tag)

Reply:
I mean, we dont need to add _darwin.go if rename works fine.